daemon/container.go

```diff
@@ -257,7 +257,7 @@ func (container *Container) Start() (err error) {
 			container.setError(err)
 			// if no one else has set it, make sure we don't leave it at zero
 			if container.ExitCode == 0 {
-				container.ExitCode = 128
+				container.ExitCode = 125
```

Comment:
I'd rather leave 128 here just to be sure we know this is the case and we can differentiate from the other codes